roles/ocp_on_libvirt/defaults/main.yml

```diff
@@ -1,9 +1,17 @@
 ---
 libvirt_image_path: /var/lib/libvirt/images
-enable_conserver: false
+bootmode: "{{ 'uefi' if enable_redfish else 'legacy' }}"
+enable_conserver: true
 enable_legacy_vga_mode: false
 do_dns_config: true
 apps_ip_address: 192.168.123.10
 api_ip_address: 192.168.123.5
 dns_vip_address: 192.168.123.6
+
+# REDFISH
+enable_redfish: false
+enable_virtualmedia: false
+redfish_cache_dir: /var/lib/dci-openshift-agent
+redfish_assisted_deploy_repo: https://github.com/redhat-partner-solutions/crucible.git
```

Comment:
You don't need this any more as the roles are now in this collection :)

Reply:
Right! thanks